ffd_info_exchange/fafsa/forms.py

```diff
@@ -12,15 +11,27 @@
 DEGREES = (('1', "First bachelor’s degree"), ('2', "Second bachelor’s degree"), ('3', "Associate degree (occupational or technical program)"), ('4', "Associate degree (general education/transfer program)"), ('5', "Certificate/diploma (occupational/technical/educational program of less than two years)"), ('6', "Certificate/diplomat (occupational/technical/educational program of at least two years)"), ('7', "Teaching credential program (nondegree)"), ('8', "Graduate or professional degree"), ('9', "Other or undecided"))
+CONSENT_TO_RETRIEVE_DATA = (('1', 'Import my 2015 tax information using the Data Retrieval Tool.'), ('0', 'Do not import my 2015 tax information — I’ll enter it manually.'))
 PARENTS_SCHOOL_COMPLETION = (('middle'), ("Middle school/junior high")), (('high'), ("High school")), (('college'), ("College or beyond")), (('not_sure'), ("I don’t know")),
 HOUSING_PLANS = (('on_campus', "On campus"), ('parents', "With parent(s)"), ('off_campus', "Off campus"))
 MARITAL_STATUS_PARENTS = (('never_married', "Never married"), ('live_together', "Unmarried and both parents living together"), ('married', "Married or remarried"), ('divorced', "Divorced or separated"), ('widowed', "Widowed"))
-TAX_COMPLETION_STATUS = (('already_filed', "They’ve already completed it"), ('will_file', "They will file it"), ('wont_file', "They’re not going to file it"))
+TAX_COMPLETION_STATUS_PARENTS = (('already_filed', "They’ve already completed it"), ('will_file', "They will file it"), ('wont_file', "They’re not going to file it"))
+TAX_COMPLETION_STATUS_STUDENT = (('already_filed', "I already completed my tax return"), ('will_file', "I will file my tax return"), ('wont_file', "I'm not going to file my tax return"))
 TAX_FILING_STATUS = (('single', "Single"), ('head', "Head of household"), ('joint', "Married — filed joint return"), ('separate', "Married — filed separate returns"), ('widowed', "Qualifying widow or widower"), ('not_sure', "I’m not sure"))
 TAX_FORM_TYPES = (('1040', "IRS 1040"), ('1040a', "IRS 1040A or 1040EZ"), ('foreign', "Foreign tax return"), ('associated', "A tax return with Puerto Rico, a U.S. territory, or a freely associated state"))
 BENEFIT_PROGRAMS = (('medicaid', "Medicaid"), ('ssi', "Supplemental Security Income (SSI)"), ('snap', "Supplemental Nutrition Assistance Program (SNAP)"), ('lunch', "Free or reduced-price school lunch"), ('tanf', "Temporary Assistance for Needy Families (TANF)"), ('wic', "Special Supplemental Nutrition Program for Women, Infants, and Children (WIC)"), ('none', "None of the above"))
 
 
+irs_data =  { 'student_return_type': '1040',
+              'student_agi': 4350,
+              'student_earned': 5153,
+              'student_is_dislocated': '0',
+              'student_received_benefits': 'snap',
+              'student_eligible_for_simpler': '0',
+              # @todo: Fill in the remainder. Start with FAFSAApplicationForm14.
```

Comment:
@micahsaul If the rest of this is working properly, I recommend picking up here and in the corresponding parts of FAFSAApplicationForm14.